src/main.rs

```diff
@@ -128,7 +128,8 @@ async fn get_pow_challenge(
             state.l1_wallet.read().balance().confirmed,
             SETTINGS.pow.min_balance,
             SETTINGS.sats_per_claim,
-        ) as u8;
+        )
+        .unwrap() as u8;
```

Comment:
Now it is my turn.
Can you please either add a comment explaining why is safe to `unwrap` here or `expect()`.